### DISCUSSION TITLE
Specifying a maxDuration in vercel.json for an Python application

### DISCUSSION THREAD

**Discussion by larsobral** [vercel/vercel#11191] [Help]
### Question

I've upgraded to a Pro plan in hosting myPython api app on Vercel. I expected that the maxDuration should have been increased from the default 10 seconds to 60 seconds (for the pro plan) but it seems I have to explicitly specify the maxDuration value in my vercel.json file. Here's my vercel.json:
{
  "version": 2,
  "builds": [
    {
      "src": "run.py",
      "use": "@vercel/python"
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "run.py"
    }
  ],
  "functions": {
    "app/*.py": {
      "maxDuration": 60
    }
  }
}

When I commit and successfully pushed to Github, Github shows an error-based notification from Vercel that says "Conflicting functions and builds Configuration" which clearly says that in my vercel.json file I can't use the "builds" and the "functions" command together; I can only use either of them.

I tried deleting the "builds" command. Upon pushing again, Vercel couldnt interprete/convert my python functions/files into serverless functions. Though I know that the build command is definitely necessary.

I really need help. How can I do the configuration in a way that I can set a maxDuration so that my API doesn't timeout after the default 10 seconds? Thanks.

**Comment by xianshui** [comment]
hi @larissavilelasobral, i encountered the same issue, have this issue been resolved?
